libgit2-sys/build.rs

```diff
@@ -27,11 +80,13 @@ fn main() {
             libgit2_vendored,\
         )"
     );
+    println!("cargo:rustc-check-cfg=cfg(libgit2_experimental_sha256)");
```

Comment:
Can this just be added to the print just above it?
It takes a comma-separated list, right?

I think that's the reason why it is formatted the way it is. The intent was to make it easy to add additional cfg values (one per line).